store/postgres/src/relational_queries.rs

```diff
@@ -529,7 +529,11 @@ impl<'a> QueryFragment<Pg> for QueryValue<'a> {
                 }
                 ColumnType::TSVector(_) => {
                     out.push_sql("to_tsquery(");
-                    out.push_bind_param::<Text, _>(s)?;
+                    let mut value = s.clone();
+                    if value.contains(" ") {
+                        value = format!("'{}'", value);
+                    }
+                    out.push_bind_param::<Text, _>(&value)?;
```

Comment:
In this code, we try to avoid allocations as much as possible since diesel will call this code 3 or 4 times every time we issue a query, even though in this case it can't be avoided entirely. It would be better to structure this as
```rust
if s.contains(" ") {
  out.push_bind_param::<Text,_>(&format!("'{}'}, s)?;
} else {
  out.push_bind_param::<Text, _>(&s)?;
}
```

Also, is containing a space the only reason why this needs to be quoted? What happens if the value contains single quotes like `"John O'Leary"`?

Reply:
I'll add a test, we probably need to escape any additional single quotes before pushing it